src/main/java/marquez/service/mappers/DatasetRowMapper.java

```diff
@@ -19,16 +19,16 @@
 import java.util.UUID;
 import lombok.NonNull;
 import marquez.common.models.NamespaceName;
-import marquez.db.models.DataSourceRow;
 import marquez.db.models.DatasetRow;
+import marquez.db.models.DatasourceRow;
 import marquez.service.models.DbTableVersion;
 
 public final class DatasetRowMapper {
   private DatasetRowMapper() {}
 
   public static DatasetRow map(
       @NonNull NamespaceName namespaceName,
-      @NonNull DataSourceRow dataSourceRow,
+      @NonNull DatasourceRow dataSourceRow,
```

Comment:
minor: shouldn't we be consistent in how we are naming datasource?

Reply:
Please see PRs #394, #387, and #384, where this is fixed.